src/jit/inlinepolicy.cpp

```diff
@@ -26,7 +26,19 @@
 
 InlinePolicy* InlinePolicy::GetPolicy(Compiler* compiler, bool isPrejitRoot)
 {
-    // For now, always create a Legacy policy.
+
+#ifdef DEBUG
+
+    // If JitInlineRandomSeed is set, use the RandomPolicy.
+    unsigned seed = JitConfig.JitInlineRandomSeed();
```

Comment:
I'm actually puzzled why the existing jit stress (at least as far as I know) doesn't have some kind of external seed value. While it is certainly somewhat stressful, it could exhibit far more variation with a combined method hash + seed like I have proposed for the random inliner.

Consider currently that JitStress=2 is still a single run per test case. With an external seed there would be 2^32 variations of JitStress=2 and we could run as little or much of this as we cared.